src/index.js

```diff
@@ -317,23 +338,27 @@ export default class RNPickerSelect extends PureComponent {
                     {this.renderTextInputOrChildren()}
                 </TouchableWithoutFeedback>
                 <Modal
+                    testID="RNPickerSelectModal"
+                    {...modalProps}
```

Comment:
Any thoughts on having this destructure happen after our defined modal props?

Having the destructure happen after is more flexible for people using the component, but I could see concerns on it causing unnecessary issues being reported.